src/Dynatrace.OpenTelemetry.Exporter.Metrics/DynatraceMetricsMapper.cs

```diff
@@ -0,0 +1,114 @@
+// <copyright company="Dynatrace LLC">
+// Copyright 2021 Dynatrace LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System.Collections.Generic;
+using OpenTelemetry.Metrics.Export;
+using DynatraceMetric = Dynatrace.MetricUtils.Metric;
```

Comment:
I think it's unfortunate that our class is called `Metric` and it clashes with the OTel one. Specially the exception. If you look at the `Dynatrace.MetricUtils` you see `MetricException` but in the trycatch we see `DynatraceMetricException`. I have to look at this alias to see what's up. 

I'd vote to open a different PR to adapt the names to use `Dynatrace` suffix to avoid clashing with OTel types.

Reply:
Agreed. :+1:

Reply:
We will do that in a follow-up PR.